packages/core/test/types.test.ts

```diff
@@ -386,3 +387,23 @@ describe('Typestates', () => {
     expect(failed).toEqual({ result: none, error: 'oops' });
   });
 });
+
+describe('interpreter', () => {
+  it('should be convertable to Rx observable', () => {
```

Comment:
an explicit test covering for changes from: https://github.com/statelyai/xstate/pull/2371/files